### PR TITLE
37.edit and delete event

### PR DESCRIPTION
**Pull request by gatilogo**
[//]: # (These are comments and are used for reference and will not show up in the PR)

# Description

[//]: # (Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change)
Integrated edit vibe event together with add vibe event

[//]: # (automatically closes given issue number)

# Proof of Concept

[//]: # (Attach Image of feature added or fix applied and working)
[//]: # (Please provide images of tests passing)
image

# Checklist

[//]: # (Add an x between square brackets to mark item as done)

- [x] Follows style guidelines and is properly commented
- [x] Code changes generate no new warnings
- [ ] ~~Explained Testing Strategy~~

### CODE REVIEW

File: app/src/main/res/layout/activity_add_edit_vibe.xml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        android:layout_height="wrap_content">
+
+        <EditText
+            android:id="@+id/reason_edit_text"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:autofillHints="false"
+            android:hint="@string/reason_hint"
+            android:inputType="text"
+            android:maxLength="20" />
```
Comment (jacobwong98) [inline app/src/main/res/layout/activity_add_edit_vibe.xml]: Need to restrict to allow only 3 words

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1,9 +1,13 @@
 <resources>
     <string name="app_name">Vybe</string>
 
+    <string name="add_vybe_name">Add Vybe</string>
```
Comment (jacobwong98) [inline app/src/main/res/values/strings.xml]: Change programmatically for Edit Vybe

File: app/src/main/res/layout/activity_add_edit_vibe.xml (hunk 1 above)
Comment (gatilogo) [reply app/src/main/res/layout/activity_add_edit_vibe.xml]: Will do in another issue

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (gatilogo) [reply app/src/main/res/values/strings.xml]: Resolved

File: app/src/main/java/com/example/vybe/AddEditVibeActivity.java [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+        outputBox = findViewById(R.id.textView);
+
+        Bundle extras = getIntent().getExtras();
+
+        if (extras != null) {
+            //TODO: set vibe and socsit dropdrowns
+            vibeEvent = (VibeEvent) extras.getSerializable("vibeEvent");
+            reasonField.setText(vibeEvent.getReason());
+
+            setTitle(getString(R.string.edit_vybe_name));
```
Comment (raysarinas) [inline app/src/main/java/com/example/vybe/AddEditVibeActivity.java]: very nice

**Pull request merged**